db/models.py

```diff
@@ -14,3 +14,35 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField("Actor", related_name="movie_actors")
+    genres = models.ManyToManyField("Genre", related_name="movie_genres")
```

Comment:
Nor critical but from checklist:
Use the correct `related_name`. In the majority of cases, the name of the model in plural will be just right.